### PR TITLE
checkaddress - unified

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

File: cs/ccxt/base/Exchange.cs [hunk 1]
```diff
@@ -511,9 +511,9 @@ public string totp(object a)
         return "";
     }
 
-    public bool checkAddress(object address)
+    public List<string> unique(object obj)
```
Comment (ttodua) [inline cs/ccxt/base/Exchange.cs:514]: this method was missing from `c#` base. though, it's only used in `checkAddress` method

File: cs/ccxt/base/Exchange.Options.cs [hunk 2]
```diff
@@ -80,6 +80,7 @@ public partial class Exchange
     public string password { get; set; }
     public string uid { get; set; }
     public string accountId { get; set; }
+    public int minFundingAddressLength { get; set; }
```
Comment (carlosmiei) [inline cs/ccxt/base/Exchange.Options.cs:83]: you declare it here but it's never initialized ?

Comment (ttodua) [reply cs/ccxt/base/Exchange.Options.cs]: ah, forgot to set to 1 (like in other langs https://github.com/ccxt/ccxt/blob/master/php/Exchange.php#L266 )

Comment (ttodua) [reply cs/ccxt/base/Exchange.Options.cs]: done.

**Pull request merged**